### PR TITLE
dependabots

### PR DESCRIPTION
**Pull request by hsorathia**
Resolves #676, resolves  #679, resolves  #686, resolves  #689 

aaa

### CODE REVIEW

**Comment by hsorathia** [comment]
o m g it WORKS

**Review by IsitaB — APPROVED** [review]

**Review by aashrav — APPROVED** [review]

File: src/Pages/MembershipApplication/membershipApplication.jsx [hunk 1]
```diff
@@ -1,23 +1,24 @@
-import React, { useState } from 'react'
-import './register-page.css'
-import MembershipPlan from './MembershipPlan.js'
-import { memberApplicationState } from '../../Enums'
-import MembershipForm from './MembershipForm'
-import ConfirmationPage from './ConfirmationPage'
-import { CSSTransition } from 'react-transition-group'
+import React, { useState } from 'react';
```
Comment (jerrylee17) [inline src/Pages/MembershipApplication/membershipApplication.jsx:1]: Why is this file changed?

File: src/Pages/Profile/admin/AdminView.jsx [hunk 2]
```diff
@@ -1,33 +1,35 @@
-import React, { useState } from "react";
+import React, { useState } from 'react';
```
Comment (jerrylee17) [inline src/Pages/Profile/admin/AdminView.jsx:1]: Why is this file changed?

**Review by jerrylee17 — CHANGES_REQUESTED** [review]
Works but some files that should not have been changed were changed

File: src/Pages/MembershipApplication/membershipApplication.jsx (hunk 1 above)
Comment (hsorathia) [reply src/Pages/MembershipApplication/membershipApplication.jsx:1]: In the bump from react-scripts 3.4.3 to 4.0.0 there were some changes made to eslint. Now, JSX files have some extra linting done to them similar to how a js file is linted. We had to update these files to meet that standard so it could build properly without throwing any errors.

See the changelog for more information:
https://github.com/facebook/create-react-app/blob/master/CHANGELOG.md

File: src/Pages/Profile/admin/AdminView.jsx (hunk 2 above)
Comment (hsorathia) [reply src/Pages/Profile/admin/AdminView.jsx:1]: see comment above

**Review by jerrylee17 — APPROVED** [review]
